app/src/Homepage/Homepage.tsx

```diff
@@ -4,8 +4,8 @@ import './Homepage.css';
 function App() {
   return (
     <div className="App">
-      <Header/>
```

Comment:
Why swap the ordering? I feel its cleaner to have Header then Sidebar since that's the ordering on the website. Makes it look more intuitive and easier to read

Reply:
Screen does in fact like this when the order is Header then Sidebar. I do not know why, but ChatGPT is telling me it's because the Sidebar is rendered after the Header, meaning it will appear on top of the Header. 

<img width="916" alt="Screen Shot 2023-10-20 at 11 40 14 PM" src="https://github.com/twang7310/rate-my-cse/assets/145406076/711efeff-8702-4fee-a005-30e7e979a99d">

If we want to preserve the site ordering, we could cut down the height of the Sidebar so it doesn't overlap the Header, and get rendered second just fine.